### ISSUE TITLE
Was the LMO dataset also trained for 50 epochs?

### ISSUE THREAD

**Issue by flashmoment** [aau-cns/poet#19]
Was the LMO dataset also trained for 50 epochs?

**Comment by tgjantos** [comment]
No, we trained PoET for the LM-O dataset for around 150 epochs, as the dataset is way smaller than YCB-V.

Best,
Thomas
